client/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "health-career-access-program-client",
   "version": "0.1.0",
   "private": true,
-  "proxy": "http://server:8080",
+  "proxy": "http://server:8081",
```

Comment:
Is there any particular reason that this was changed?

Reply:
@will-freshworks I realized this might not end up in your inbox unless I ping you, since I created the PR!

Reply:
Also curious about this.

Reply:
I needed to change the port because Keycloak also uses 8080. Due to the way the docker-compose is set-up you cannot have server and keycloak sharing the same port. This is because when the browser accesses keycloak it goes to `keycloak.local.freshworks.club:8080` which resolves to `localhost:8080`, which would clash with the server unless we moved it. We can't rebind the keycloak port because when the `server` accesses keycloak it also has to use `keycloak.local.freshworks.club:8080`  which resolves to `keycloak:8080` (inside the docker network because of the `alias` and we cannot rebind ports within the docker network, rebinding only works for ports exposed to the external host machine.

This is a little complex I realise. I might do a swamp talk on it...

Reply:
Oh, I actually already did a swamp talk on this I remember, it was my first one I think